Specs/DateTime/German/DateTimeModel.json

```diff
@@ -2744,6 +2744,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
+    "NotSupported": "dotnet",
```

Comment:
This can't happen.

Reply:
A similar new test case for which dotnet is supported is added at line 5622, as dotnet output is modified.
particular dotnet change is not reflected to other languages.

Reply:
Not sure what you mean by this, but like I said, we can't have cases that worked before now not working. This is a blocker. Sorry.